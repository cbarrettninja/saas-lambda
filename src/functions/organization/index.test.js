describe('index', () => {
  it('executes as expected', () => {
    const statusCode = 200;
    expect(statusCode).toMatchSnapshot();
  });
});
