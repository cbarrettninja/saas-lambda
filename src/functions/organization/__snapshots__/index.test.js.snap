// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index executes as expected 1`] = `200`;
